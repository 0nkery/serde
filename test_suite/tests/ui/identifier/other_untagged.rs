#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
#[serde(untagged)]
enum F {
    #[serde(other)]
    Other,
}

fn main() {}
