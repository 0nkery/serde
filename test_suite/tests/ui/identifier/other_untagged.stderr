error: #[serde(other)] cannot appear on untagged enum
 --> $DIR/other_untagged.rs:7:5
  |
7 | /     #[serde(other)]
8 | |     Other,
  | |_________^

error: aborting due to previous error

